scss/_accordion.scss

```diff
@@ -44,26 +64,26 @@
 
```

Comment:
For the accordions I was hesitant to add CSS vars everywhere since it's not done in Bootstrap. Tried https://github.com/twbs/bootstrap/pull/36148 to see what's the direction to follow on that. Wait & see.

Reply:
Bootstrap PR has been merged so I'll use CSS vars everywhere when I'll gather this new commit from Bootstrap. I let the Scss vars in this PR for the accordions as it is if you don't mind. If OK with you I let you resolve all the corresponding accordions comments.